src/main/java/io/managed/services/test/dataplane/FleetshardUtils.java

```diff
@@ -0,0 +1,77 @@
+package io.managed.services.test.dataplane;
+
+import io.fabric8.kubernetes.api.model.KubernetesResourceList;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.dsl.MixedOperation;
+import io.fabric8.kubernetes.client.dsl.Resource;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.managed.services.test.k8.managedkafka.v1alpha1.ManagedKafka;
+import io.managed.services.test.k8.managedkafka.v1alpha1.ManagedKafkaAgent;
+import lombok.extern.log4j.Log4j2;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Log4j2
+public class FleetshardUtils {
+
+    private static final String BF2_DOMAIN = "bf2.org/";
+    private static final String PROFILE_QUOTA_CONSUMED = BF2_DOMAIN + "kafkaInstanceProfileQuotaConsumed";
+    private static final String PROFILE_TYPE = "bf2.org/kafkaInstanceProfileType";
```

Comment:
These constants are already declared in https://github.com/bf2fc6cc711aee1a0c2a/e2e-test-suite/blob/825e7594e1135a566dc73396c397bdef1c95bc08/src/main/java/io/managed/services/test/k8/managedkafka/v1alpha1/ManagedKafka.java#L34-L42

Reply:
yes but these will be removed (it is just temporal solution how to include managedkafka)

Reply:
Once they are removed, you'll get them from the imported jar i.e these fields will remain public once there is a published managedkafka jar.